pkg/proxy/proxy.go

```diff
@@ -32,6 +32,9 @@ import (
 	"github.com/nordix/meridio/pkg/networking"
 )
 
+const dstPrefix = "-dst"
```

Comment:
ambiguous names considering the local variables used in SetIPContext() to store return of Allocate()